### PR TITLE
Improve descriptions and layouts, schema adjustments

### PR DESCRIPTION
**Pull request by Maurier**

### CODE REVIEW

File: dspback/schemas/zenodo/schema.json [hunk 1]
```diff
@@ -324,20 +324,14 @@
     "references": {
```
Comment (Maurier) [inline dspback/schemas/zenodo/schema.json:324]: According to Zenodo API documentation, the field 'references' is an array of strings: https://developers.zenodo.org/#representation

I've adjusted our schema accordingly.

File: .env [hunk 2]
```diff
@@ -48,7 +48,7 @@ EARTHCHEM_DELETE_URL=https://staging-ecl-api.earthchem.org/v2/records/%s
 EARTHCHEM_FILE_CREATE_URL=https://staging-ecl-api.earthchem.org/v2/files/%s/uploadFile
 EARTHCHEM_FILE_DELETE_URL=https://staging-ecl-api.earthchem.org/v2/files/%s/%s
 EARTHCHEM_FILE_READ_URL=https://staging-ecl-api.earthchem.org/v2/files/%s
-EARTHCHEM_VIEW_URL=https://test-ecl.earthchem.org/view.php?id=%s
+EARTHCHEM_VIEW_URL=https://test-ecl.earthchem.org/my/my_view.php?id=%s
```
Comment (Maurier) [inline .env:51]: Note that I've updated this URL in the environment file and will probably need to be applied to the environment files of other deployments.

Comment (sblack-usu) [reply .env:51]: Updated this in all environments

**Review by sblack-usu — APPROVED** [review]
I regenerated the schemas as well as added a default empty list to the external contributors property.

**Pull request merged**